forte_wrapper/vader/sentiment_analysis.py

```diff
@@ -51,24 +51,36 @@ def __init__(self):
 
     def initialize(self, resources: Resources, configs: Config):
         # pylint: disable=unused-argument
+        super().initialize(resources, configs)
         self.sentence_component = configs.get('sentence_component')
 
     def _process(self, input_pack: DataPack):
-        for sentence in input_pack.get(entry_type=Sentence,
-                                       components=self.sentence_component):
-            scores = self.analyzer.polarity_scores(sentence.text)
-            sentence.sentiment = scores
+        path_str, module_str = self.configs.entry_type.rsplit('.', 1)
+
+        mod = importlib.import_module(path_str)
+        entry = getattr(mod, module_str)
```

Comment:
Don't do this, the pack can directly get item using string. `input_pack.get("ft.onto.base_ontology.Sentence")